src/index.js

```diff
@@ -118,9 +118,13 @@ async function main({
         body: e.message,
       };
     }
-    log.error('Unhandled error', e, e.stack);
+    log.error('Unhandled error', e,
+    /* istanbul ignore next */
+      e.stack || 'no stack');
     return {
-      body: e.message,
+      body:
+      /* istanbul ignore next */
+      e.message || 'no message',
```

Comment:
same here. if `e` is undefined, this throws.

Reply:
```suggestion
      (e && e.message) || 'no message',
```